### ISSUE TITLE
SelectField SVG accessibility

### ISSUE THREAD

**Issue by aleshh** [aws-amplify/amplify-ui#5253]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have tried disabling all browser extensions or using a different browser
- [X] I have tried deleting the node_modules folder and reinstalling my dependencies
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Other

### How is your app built?

vite

### What browsers are you seeing the problem on?

Firefox

### Please describe your bug.

SelectField has a nifty drop-down icon SVG. This gets flagged in accessibility scan because there is no accessible name. I think the SVG should have `aria-hidden="true"`, perhaps as an option. 

Work-around is to pass in an custom icon with the appropriate props.

### What's the expected behaviour?

SelectField should have `aria-hidden="true"` as an option or maybe even default.

### Help us reproduce the bug!

This is visible on the Amplify docs demo.

### Code Snippet

```typescript
// Put your code below this line.

```
